source/seedsource/urls.py

```diff
@@ -2,7 +2,7 @@
 from django.views.generic.base import TemplateView
 from rest_framework.routers import DefaultRouter
 
-from seedsource.views import TransferLimitViewset, SeedZoneViewset, GeneratePDFView, RegionsView
+from seedsource.views import TransferLimitViewset, SeedZoneViewset, GeneratePDFView, GeneratePowerPointView, RegionsView
```

Comment:
Looks like this line is running over 120 columns. Consider doing `from seedsource import views` and then `view.GeneratePDFView`, etc.